app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java

```diff
@@ -211,23 +233,25 @@ public void onSaveInstanceState(@NonNull Bundle outState) {
     public void onViewStateRestored(@Nullable Bundle savedInstanceState) {
         super.onViewStateRestored(savedInstanceState);
         if(null!=savedInstanceState){
-            lastVisibleItemID =savedInstanceState.getString(VISIBLE_ITEM_ID, null);
```

Comment:
not restoring the state?

Reply:
The scrolled state of the list is still being retained. I am not sure how this variable was being used earlier.